torchci/components/WorkflowBox.tsx

```diff
@@ -174,7 +183,22 @@ export default function WorkflowBox({
             ></input>
             <input type="submit" value="Search"></input>
           </form>
+          <div
+            style={{
+              // Ensures elements after this div are actually below it (due to float)
+              clear: "both",
+            }}
+          ></div>
           {searchString && <div>{searchRes.info}</div>}
+          <div style={{ margin: ".5em 0em" }}>
+            <span
+              onClick={() => {
+                setWide(!wide);
+              }}
+            >
+              {wide ? "Hide Additional Test Info" : "Show Additional Test Info"}
+            </span>
+          </div>
```

Comment:
Can you format this to look like a button? It's not at all obvious that this is clickable right now:

<img width="724" alt="image" src="https://github.com/pytorch/test-infra/assets/4468967/e42c2f31-0c42-42e4-80f7-3ade67d4b4c2">